### PR TITLE
refactor: replace missing process.env with envConfig/gwConfig

### PR DESCRIPTION
**Pull request by RetricSu**
we have some missing config still using `process.env.` instead of `envConfig` or `gwConfig`. this pr refactor such code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ ENABLE_CACHE_ETH_CALL=<optional, enable eth_call cache, default to false>
 LOG_LEVEL=<optional, allowed value: `debug` / `info` / `warn` / `error`, default to `debug` in development, and default to `info` in production>
 LOG_FORMAT=<optional, allowed value: `json`>
 MAX_SOCKETS=<optional, max number of httpAgent sockets per host for web3 connecting to godwoken, default to 10>
+WEB3_LOG_REQUEST_BODY=<optional, boolean, if true, will log request method / body, default to false>
+PORT=<optional, the api-server running port, default to 3000>
```
Comment (keroro520) [inline README.md:31]: I prefer using 8024 as the default port.  Grafana uses 3000 as the default port.
The default port should be the same as https://github.com/RetricSu/godwoken-kicker/blob/compatibility-changes/docker/docker-compose.yml#L127

File: README.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ ENABLE_CACHE_ETH_CALL=<optional, enable eth_call cache, default to false>
 LOG_LEVEL=<optional, allowed value: `debug` / `info` / `warn` / `error`, default to `debug` in development, and default to `info` in production>
 LOG_FORMAT=<optional, allowed value: `json`>
 MAX_SOCKETS=<optional, max number of httpAgent sockets per host for web3 connecting to godwoken, default to 10>
+WEB3_LOG_REQUEST_BODY=<optional, boolean, if true, will log request method / body, default to false>
```
Comment (keroro520) [inline README.md:30]: we can define a new log-level, "trace", to support tracing logs.

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (keroro520) [reply README.md:30]: It is out of this PR.

File: README.md (hunk 1 above)
Comment (RetricSu) [reply README.md]: updated

**Review by keroro520 — APPROVED** [review]

**Review by Flouse — APPROVED** [review]

**Pull request merged**